test/test_serialization.py

```diff
@@ -0,0 +1,85 @@
+
+from dynatrace.opentelemetry.metrics.export import _get_histogram_min, _get_histogram_max
+
+
+def create_histogram(explicit_bounds: List[int], bucket_counts: List[int], sum: Union[float, int]):
+    start_time = 1619687639000000000
+    end_time = 1619687639000000000
+    return Histogram(bucket_counts=bucket_counts,
+                     explicit_bounds=explicit_bounds,
+                     sum=sum,
+                     min=+math.inf,
+                     max=-math.inf,
+                     aggregation_temporality=AggregationTemporality.DELTA,
+                     time_unix_nano=end_time,
+                     start_time_unix_nano=start_time)
+
+
+class TestMin(unittest.TestCase):
+    def test_get_min(self):
+        # A value between the first two boundaries.
+        self.assertEqual(1,
+                         _get_histogram_min(create_histogram([1, 2, 3, 4, 5], [0, 1, 0, 3, 0, 4], 10.234)))
+        # lowest bucket has value, use the first boundary as estimation instead of -Inf
+        self.assertEqual(1,
+                         _get_histogram_min(create_histogram([1, 2, 3, 4, 5], [1, 0, 0, 3, 0, 4], 10.234)))
+        # lowest bucket (-Inf, 1) has values, mean is lower than the lowest bucket bound and smaller than the sum
+        self.assertAlmostEqual(0.234 / 3,
+                               _get_histogram_min(create_histogram([1, 2, 3, 4, 5], [3, 0, 0, 0, 0, 0], 0.234)),
+                               delta=0.001)
+        # lowest bucket (-Inf, 0) has values, sum is lower than the lowest bucket bound
+        self.assertAlmostEqual(-25.3,
+                               _get_histogram_min(create_histogram([0, 5], [3, 0, 0], -25.3)),
+                               delta=0.001)
+        # no bucket has a value
+        self.assertAlmostEqual(10.234,
+                               _get_histogram_min(create_histogram([1, 2, 3, 4, 5], [0, 0, 0, 0, 0, 0], 10.234)),
+                               delta=0.001)
+        # just one bucket from -Inf to Inf, calc the mean as min value.
+        self.assertAlmostEqual(2.2,
+                               _get_histogram_min(create_histogram([], [4], 8.8)),
+                               delta=0.001)
+        # just one bucket from -Inf to Inf, with a count of 1
+        self.assertAlmostEqual(1.2,
+                               _get_histogram_min(create_histogram([], [1], 1.2)),
+                               delta=0.001)
+        # only the last bucket has a value (5, +Inf)
+        self.assertAlmostEqual(5,
+                               _get_histogram_min(create_histogram([1, 2, 3, 4, 5], [0, 0, 0, 0, 0, 1], 10.234)),
+                               delta=0.001)
+
+
+class TestMax(unittest.TestCase):
+    def test_get_max(self):
+        #  A value between the last two boundaries.
+        self.assertEqual(5,
+                         _get_histogram_max(create_histogram([1, 2, 3, 4, 5], [0, 1, 0, 3, 0, 4], 10.234)))
```

Comment:
Alternatively to changing the comment you could also change the buckets here 
```suggestion
                         _get_histogram_max(create_histogram([1, 2, 3, 4, 5], [0, 1, 0, 3, 2, 0], 10.234)))
```

Reply:
I did that instead. [bccf14e](https://github.com/dynatrace-oss/opentelemetry-metric-python/pull/23/commits/bccf14e4735011c138a80bdccbc2eb2b5297b03f). I copied that from the min tests on accident. 

Thanks for catching this! :slightly_smiling_face: It can be super hard to see these errors after some time. :sweat: